app/src/main/java/com/xinto/opencord/domain/Mentionable.kt

```diff
@@ -1,5 +1,5 @@
-package com.xinto.opencord.domain.model

```

Comment:
Why are the classes outside of the models package? IMO domain itself indicates the UI data, which can consist of other provider classes not including the data models. I think even the store package should be inside the domain package

Reply:
1. store isnt domain
2. there wasn't anything left in `domain` so i moved it up a level
3. realistically there isn't much thats going to be there

Reply:
what do you mean by runtime logic? repositories also have logic for converting data but they are part of domain

Reply:
stores have caching and fetch logic
repositories are just a one-way "lambda"

Reply:
eh fine